tpcx_bb/cluster_configuration/bsql-cluster-startup.sh

```diff
@@ -0,0 +1,101 @@
+#IB, NVLINK, or TCP
+CLUSTER_MODE=$1
+USERNAME=$(whoami)
+
+MAX_SYSTEM_MEMORY=$(free -m | awk '/^Mem:/{print $2}')M
+DEVICE_MEMORY_LIMIT="25GB"
+POOL_SIZE="30GB"
+
+# Fill in your environment name and conda path on each node
+TPCX_BB_HOME="/home/$USERNAME/shared/tpcx-bb"
+CONDA_ENV_NAME="rapids-tpcx-bb"
+CONDA_ENV_PATH="/home/$USERNAME/conda/etc/profile.d/conda.sh"
+
+# TODO: Unify interface/IP setting/getting for cluster startup
+# and scheduler file
+INTERFACE="ib0"
+
+# TODO: Remove hard-coding of scheduler
+SCHEDULER=$(hostname)
+SCHEDULER_FILE=$TPCX_BB_HOME/tpcx_bb/cluster_configuration/example-cluster-scheduler.json
+LOGDIR="/tmp/tpcx-bb-dask-logs/"
+WORKER_DIR="/tmp/tpcx-bb-dask-workers/"
+
+# Purge Dask worker and log directories
+rm -rf $LOGDIR/*
+mkdir -p $LOGDIR
+rm -rf $WORKER_DIR/*
+mkdir -p $WORKER_DIR
+
+# Purge Dask config directories
+rm -rf ~/.config/dask
+
+# Activate conda environment
+source $CONDA_ENV_PATH
+conda activate $CONDA_ENV_NAME
+
+# Check for and set BlazingContext parameters as environment variables
+if [ -z "$BLAZING_ALLOCATOR_MODE" ]
+then
+      export BLAZING_ALLOCATOR_MODE="existing"
+fi
+
+# Check for and set BlazingContext config_options as environment variables
+if [ -z "$JOIN_PARTITION_SIZE_THRESHOLD" ]
+then
```

Comment:
You actually dont need to check the config_options environment variables here, if later when you create `config_options` you also have a default there.

Reply:
Makes sense, removed.